Sources/CALayer+ContentsGravity.swift

```diff
@@ -1,17 +1,14 @@
-//
-//  CALayer+ContentsGravity.swift
-//  UIKit
-//
-//  Created by Geordie Jay on 18.02.18.
-//  Copyright © 2018 flowkey. All rights reserved.
-//
-
-
-extension CALayer {
-    internal enum ContentsGravity: String {
-        case left, center, right, top, bottom, topLeft, topRight
-        case resize, resizeAspectFill, resizeAspectFit = "resizeAspect"
-    }
+/*
+ * Apples CALayerContentsGravity implementation is based on a struct
+ * with a raw representable because of backwards compatibility
+ * We implemented it with an enum, which can be used the
+ * same way as Apples CALayerContentsGravity
+*/
+public enum CALayerContentsGravity: String {
```

Comment:
In an ideal world we'd implement the fallback to `.center` for an unknown value here, like I'm pretty sure is still the case with Apple UIKit. But since it's not documented I don't think it's reasonable for anyone to rely on this behaviour. So I'm fine for this to go in as is!